liquibase-standard/src/main/java/liquibase/parser/core/yaml/YamlChangeLogParser.java

```diff
@@ -101,12 +102,17 @@ private Map parseYamlStream(String physicalChangeLogLocation, Yaml yaml, InputSt
     private void loadChangeLogParametersFromFile(ChangeLogParameters changeLogParameters, ResourceAccessor resourceAccessor, DatabaseChangeLog changeLog, Map property, ContextExpression context, Labels labels, Boolean global) throws IOException, LiquibaseException {
         Properties props = new Properties();
         Boolean relativeToChangelogFile = (Boolean) property.get("relativeToChangelogFile");
+        Boolean errorIfMissing = (Boolean) property.get("errorIfMissing");
```

Comment:
Are there other changelog parser classes (like JSON??) that would also need this update?

CC @rberezen @MalloD12

Reply:
My understanding is that there is not, because for example, JSON changelog parser extends from this one (YamlChangeLogParser), so I think with this one we are ok.